src/CLI/Utilities/ArgumentHandler.cs

```diff
@@ -9,12 +10,88 @@ namespace CLI.Utilities
 {
     internal class ArgumentHandler
     {
-        internal async Task RunScripts(ArgumentParser argumentParser)
+        public RootCommand BuildRootCommand()
         {
+            var singleBoxDeployOption = new Option<bool>(
+            name: "--single-box-deploy",
+            getDefaultValue: () => false,
+            description: "Prepares and installs hub and spoke environments"
```

Comment:
Let's be a bit more descriptive here, I want the user to know that this deploys to the current machine and does not need additional VMs.
```suggestion
            description: "Prepares and installs hub and spoke environments on the current machine"
```